### PR TITLE
PLATZ view

### PR DESCRIPTION
**Pull request by yes-ho**
This function is a view that you can surf the PLATZ page.
This is permitted by ONLY VD member.
Review PLZ..

### CODE REVIEW

File: project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/OpenPlatzViewHandler.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		try {
+			IViewPart platzPart = EclipseUtil.findView(PlatzView.ID);
+			final PlatzView platzview = (PlatzView) platzPart;
+			StringBuilder url = new StringBuilder();
+			url.append(DexterConfig.PLATZ_API_URL);
+			platzview.setUrl(url.toString());
+			EclipseUtil.showView(PlatzView.ID);
+
+		} catch (DexterRuntimeException e) {
+			MessageDialog.openError(part.getSite().getShell(), "PLATZ Error", "Cannot open the PLATZ View");
```
Comment (Minjung-Baek) [inline project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/OpenPlatzViewHandler.java]: In the eclipse documentation, if an exception occured, it should throw the ExecutionException.
 Please test which one is better between opening Message Dialog or throw ExecutionException.
 Read this: http://help.eclipse.org/juno/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fcore%2Fcommands%2FIHandler.html
 (If you can test it next version)

File: project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/view/PlatzView.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	@Override
+	public void setFocus() {
+	}
+	
+	public void isAlive(){
+		browser.addProgressListener(new ProgressListener() {
+			
+			@Override
+			public void completed(ProgressEvent event) {
+			System.out.println(event);
```
Comment (Minjung-Baek) [inline project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/view/PlatzView.java]: Please remove test code

File: project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+public class MonitorForPlatzKeywordFile extends Job implements IDexterHomeListener {
+	public static long LAST_CONF_CHANAGED_TIME = -1;
+	private final static int MONITOR_DELAY = 500;
+	private final static int SERVER_TIMEOUT = 500;
+	private File keywordFile;
+
+	public MonitorForPlatzKeywordFile() {
+		super("Monitoring target keyword in sourceinsight");
+		try {
+			if (!InetAddress.getByName(DexterConfig.PLATZ_DOMAIN).isReachable(SERVER_TIMEOUT)) {
```
Comment (Minjung-Baek) [inline project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java]: Please confirm about the SERVER_TIMEOUT variable.
Modification is unnecessary if it is enough for a value now.
https://community.oracle.com/thread/1150954?tstart=0

File: project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+				initPlatzKeywordFile();
+				DexterConfig.getInstance().addDexterHomeListener(this);
+			}
+		} catch (UnknownHostException e) {
```
Comment (Minjung-Baek) [inline project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java]: If you already know developer is not VD person, this is the unnecessary code.

Comment (yes-ho) [inline project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java]: I think so. I will remove this code.

File: project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java (hunk 3 above)
Comment (yes-ho) [inline project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/job/MonitorForPlatzKeywordFile.java]: After Enter Ping Command, average Connetion Time is 0 ms.
Therefore I think SERVER_TIMEOUT variable is enough to confirm connetion.

File: project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/view/PlatzView.java (hunk 2 above)
Comment (yes-ho) [inline project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/view/PlatzView.java]: Yes, I will remove this unused code.

**Pull request merged**